about.html

```diff
@@ -0,0 +1,96 @@
+
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" >
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" >
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" >
+    <title>Elearnig Website 2022, 2023</title>
+    <link rel="stylesheet" href="./style.css" >
+    <script defer src="./script.js"></script>
+    <script src="https://kit.fontawesome.com/10729b3e67.js" crossorigin="anonymous"></script>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0/css/all.min.css">
+  </head>
+  <body>
```

Comment:
Great job @Yordinia 🤝

- [ ] Kindly change your about section and make sure it looks like what was provided in the design. Make sure this is reflected both in the desktop and the mobile sreens.
- Your design
![Your About](https://user-images.githubusercontent.com/63917725/205319756-9dc299ca-f754-481f-917a-1acee858eeff.PNG)
- Given design
![Given About](https://user-images.githubusercontent.com/63917725/205319857-b2e7fe6d-2c4c-4ca0-9041-83e48531b6e5.PNG)